### PR TITLE
Sonar Props added

### PR DESCRIPTION
**Pull request by lgriffin**
Props added

### CODE REVIEW

**Comment by fheng-build-servers** [comment]
Build finished. No test results found.

**Comment by wtrocki** [comment]
Retest this please

**Comment by fheng-build-servers** [comment]
Build finished. No test results found.

**Comment by fheng-build-servers** [comment]
Test comment.

File: sonar-project.properties [hunk 1]
```diff
@@ -0,0 +1,10 @@
+sonar.projectKey=fh-logger
+sonar.projectName=fh-logger-nightly-master
+sonar.projectVersion=0.4.1
+
+sonar.sources=./lib
+sonar.tests=./test
+sonar.language=js
+sonar.javascript.lcov.reportPath=./coverage/lcov.info
```
Comment (wtrocki) [inline sonar-project.properties:8]: Sonar cannot find this file and places comment. We would need to make sure that unit tests and coverage file is regenerated.

Comment (lgriffin) [inline sonar-project.properties:8]: General note here: We can of course investigate and make relevant JIRAs one we have the builds consistent. The above config file is a template for the core configuration that should be included so the coverage path could be wrong (again a general note). 

In the above case the file is present it is just empty. There is a cobertura report there however.Something to research further down the line as to why it's not generating or if we can use an alternative.

**Comment by wtrocki** [comment]
Retest this please

**Comment by fheng-build-servers** [comment]
Build finished. No test results found.

**Comment by fheng-build-servers** [comment]
Build finished. No test results found.

**Comment by danbev** [comment]
Retest this please

**Comment by fheng-build-servers** [comment]
Build finished. No test results found.

**Comment by danbev** [comment]
Retest this please

**Comment by fheng-build-servers** [comment]
Build finished. No test results found.

**Comment by danbev** [comment]
Retest this please

**Comment by danbev** [comment]
@lgriffin I've update your PR with a commit to include the coverage. 

I also added the engineering team which was missing from the fh-logger github repository. This team includes`fheng-build-servers`` which I believe was the issue with the communication between Jenkins and the fh-logger github repo. Can you take a look and if all looks alright I'll merge this.

**Comment by lgriffin** [comment]
Hey @danbev thanks for checking this out -- we thought it was a global permissions issue but that makes sense (it might have resolved another issue for us as well!). 

The Sonar props look okay from my perspective, if you wish to merge it I can run a manual test ahead of tonight and make any tweaks necessary.

**Comment by lgriffin** [comment]
Analysis successfully ran, including line coverage report.

**Pull request merged**